config/metadata.yaml

```diff
@@ -0,0 +1,16 @@
+# maps release series of major.minor to cluster-api contract version
+# the contract version may change between minor or major versions, but *not*
+# between patch versions.
+#
+# update this file only when a new major or minor version is released
+apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
+releaseSeries:
+  - major: 0
+    minor: 3
+    contract: v1beta1
+  - major: 0
+    minor: 2
+    contract: v1beta1
+  - major: 0
+    minor: 1
+    contract: v1beta1
```

Comment:
I guess the `metadata.yaml` file is required due to the CAPI providers/addons contract, isn't it?

Reply:
yes, as you know(https://github.com/clastix/cluster-api-control-plane-provider-kamaji/pull/17), that is for clusterctl.

Reply:
I'm a bit puzzled about the three items, here.
Shouldn't we just address the first one (`{"major": 0, "minor": 3, "contract": "v1beta1"}`) since we're going to release this for Kamaji > v0.3.x?

Reply:
Yes, you are right. I needed that for my test.